src/main/java/core/basesyntax/model/CreateStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+import java.time.LocalDate;
+
+public class CreateStrategyImpl implements CreateStrategy {
+    @Override
+    public Fruit createFruit(String fruitType, LocalDate expirationDate) {
+        if (fruitType.equalsIgnoreCase(Banana.class.getSimpleName())) {
+            return new Banana(expirationDate);
+        } else if (fruitType.equalsIgnoreCase(Apple.class.getSimpleName())) {
+            return new Apple(expirationDate);
+        } else if (fruitType.equalsIgnoreCase(Orange.class.getSimpleName())) {
+            return new Orange(expirationDate);
```

Comment:
this code isn't flexible, cause adding/removing fruits will require changes in this class